adoc/config/rouge/lib/rouge/lexers/sycl.rb

```diff
@@ -180,76 +180,6 @@ class Sycl < Cpp
         wait_and_throw
       )
 
-      # Generic types used in SYCL pseudo code descriptions like Gen,
-      # SGen, GenVec...
-      sycl_generic_types = %w(
```

Comment:
I suggest keeping this but empty, as this infrastructure could be used in the future instead of diving again into which Rouge feature we could use for a new kind of keywords in SYCL Next.
For example if we introduce some `concept` in SYCL Next.

Reply:
Good idea.  I restored the rouge support in 904fb547cee35f3f6c61d226b2c1adee81dc6df0.

Reply:
Oh, thanks for noticing this!  I forgot to push that commit to the branch corresponding to this PR.  It should be there now.

The branches are a little complicated with all these stacked PRs.